conf/variant/arp-qtauto/bblayers.conf.sample

```diff
@@ -0,0 +1,14 @@
+# These are needed for bitbake to pick up the includes
+BSPDIR := "${TOPDIR}/.."
+BBPATH := "${TOPDIR}:${BSPDIR}/sources/meta-pelux"
+
+require conf/variant/common/bblayers.conf
+require conf/variant/common/bblayers-qtauto.conf
+
+BBLAYERS += "\
+    ${BSPDIR}/sources/meta-pelux/meta-intel-extras  \
+    ${BSPDIR}/sources/meta-intel                    \
+    ${BSPDIR}/sources/meta-arp                      \
+"
+
```

Comment:
Just to be picky... I would try to avoid empty lines that don't carry meaning. (empty lines that separate logical blocks are great, but a couple of empty lines like this aren't of any use)
No big deal though :)

Reply:
removed empty lines